src/mame/includes/pc6001.h

```diff
@@ -118,6 +118,7 @@ class pc6001_state : public driver_device
 
 	emu_timer *m_timer_irq_timer;
 	uint8_t *m_video_ram;
+	std::unique_ptr<uint8_t[]> m_video_ram_alloc;
```

Comment:
Would it make more sense to call this `m_video_ram` and call the other thing `m_video_base` or something?